packages/api-codec/src/Metadata.ts

```diff
@@ -17,12 +19,12 @@ import U16 from './U16';
 // end and work up. (Just so we don't use before definition)
 
 export class EventMetadata extends Struct {
-  constructor () {
+  constructor (value?: any) {
```

Comment:
We can now instantiate all of these, it aligns with all the other codec classes apart for just being useful